pkg/spdx/format.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package spdx
+
+import (
+	"bytes"
+	"strings"
+
+	// TODO: These should be removed once version v0.4.0 is released
+	// in https://github.com/spdx/tools-golang.
+	//
+	// This fork is required to prevent error "got unknown checksum type SHA512".
+	//
+	// See also:
+	// - https://github.com/kubernetes-sigs/bom/pull/104
+	// - https://github.com/spdx/tools-golang/pull/139
+	// - https://github.com/spdx/tools-golang/issues/96
```

Comment:
Tried following @ianling 's advice to import the fork, but that doesn't work without replace statement due to:
```
$ go mod tidy
go: github.com/ion-channel/tools-golang@v0.0.0-20220425222917-af3d04c69209: parsing go.mod:
	module declares its path as: github.com/spdx/tools-golang
	        but was required as: github.com/ion-channel/tools-golang
```

So I forked the fork into a new org called "this-is-a-fork-remove-me-asap", updated the module name, cut a release, and added this TODO